shared/src/main/kotlin/com/egm/stellio/shared/util/QueryUtils.kt

```diff
@@ -155,7 +155,10 @@ private fun transformQQueryToSqlJsonPath(
         """
         jsonb_path_exists(#{TARGET}#,
             '$.$jsonAttributePath.**{0 to 2}."$JSONLD_VALUE" ? (@ $operator ${'$'}value)',
-            '{ "value": $value }')
+            '{ "value": ${value.replace(
+            "'",
+            "''"
+        )}}')
```

Comment:
- the indentation is weird
- there are already some extension functions used here (like `String.quote()`), cleaner to add a new one for this escaping